src/runtime/auto-prompt-manager.ts

```diff
@@ -906,10 +958,10 @@ export class AutoPromptManager {
       this.storeCompletion(action!);
     } else if (GENERIC_COMPLETION_EVENTS.includes(event)) {
       if (this.isContribution_()) {
-        this.storeCompletion(TYPE_CONTRIBUTION);
+        this.storeCompletion(InterventionType.TYPE_CONTRIBUTION);
```

Comment:
Is this going to impact frequency capping, particularly how the keys are written? The current tests setup frequency capping using raw strings: https://github.com/subscriptions-project/swg-js/blob/main/src/runtime/auto-prompt-manager-test.js#L213

As the current implementation, I am not 100% sure if the tests cover this conversion of `InterventionType` to the string used as the localstorage key.

Reply:
These are enum strings, so I don't think it should impact how its written unless someone changes the enum value. Would it be better if disentangled the article names from the storage names?

Reply:
Actually, it seems like the enum -> string conversion _is_ being tested via the set expectation: https://github.com/subscriptions-project/swg-js/blob/main/src/runtime/auto-prompt-manager-test.js#L4438